shotover-proxy/src/message/mod.rs

```diff
@@ -281,13 +285,12 @@ pub enum Value {
     NamedRows(Vec<HashMap<String, Value>>),
     Document(HashMap<String, Value>),
```

Comment:
Should `Document` and `Map` be merged?

Reply:
That would make sense as a Document is a subset of Map

Reply:
Will follow up with this in a later PR when I have a better idea of what parsing and storing a cassandra map into value will look like.